src/helpers/index.ts

```diff
@@ -1,16 +1,67 @@
 import type {
   BackupKeyRecoveryTransansaction,
   FormattedOfflineVaultTxInfo,
+  RecoverParams,
 } from '@bitgo/abstract-utxo';
+import type { Chain, Hardfork } from '@ethereumjs/common';
 
 const GWEI = 10 ** 9;
 
+export async function recover(
+  coin: string,
+  token: string | undefined,
+  params: RecoverParams & {
+    rootAddress?: string;
+    gasLimit?: number;
+    gasPrice?: number;
+    eip1559?: {
+      maxFeePerGas: number;
+      maxPriorityFeePerGas: number;
+    };
+    replayProtectionOptions?: {
+      chain: typeof Chain[keyof typeof Chain];
+      hardfork: `${Hardfork}`;
+    };
+    walletContractAddress?: string;
+    tokenAddress?: string;
+  }
+) {
+  let recoverPayload;
+  if (token) {
+    try {
+      recoverPayload = await window.commands.recover(token, params);
+    } catch (e) {
+      recoverPayload = await window.commands.recover(coin, params);
+    }
+  } else {
+    recoverPayload = await window.commands.recover(coin, params);
+  }
+  return recoverPayload;
```

Comment:
don't do this, just return inside the block because the let recoverPayload is `any` which is wrong.